app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[1] == command_list[2]:
+        pass
+    elif command_list[0] == "cp" and len(command_list) == 3:
```

Comment:
len(command_list) == 3 shoud be done at the very beginning just after split, because if it's less than 3 - you will not get to this line and the script will be crashed ad 3rd line.